app/assets/javascripts/admin/addon/controllers/admin-site-settings.js

```diff
@@ -114,6 +137,17 @@ export default class AdminSiteSettingsController extends Controller {
     );
   }
 
+  fuzzySearch(query, item) {
+    const strippedQuery = query.replace(/[^a-z0-9]/gi, "");
+
+    if (strippedQuery.length > 2) {
+      const regex = new RegExp(strippedQuery.split("").join(".*"), "i");
```

Comment:
Looks like we're rebuilding this regex for every site settings? 🤔  Should we instead be doing it once and then `test` it against all the site settings?

Reply:
oh yeah ... will pull it out